### ISSUE TITLE
-M simulator parameter

### ISSUE THREAD

**Issue by evgisa** [lykaust15/DeepSimulator#12]
Hi, I'd like to know what would be the main difference between using the context-dependent and the context-independent simulator. 
After reading the paper I understood that the main advantage of DeepSimulator over the rest of nanopore simulators is its context-dependent model. So, I was wondering if there is any reason why in deep_simulator.sh the default value of -M is context-independent.
Thank you in advance!

**Comment by lykaust15** [comment]
Hi, thank you very much for asking this!

Here is the detailed reason for your question.
1. When we started this project, the Nanoporetech has not released the 6mer table, so we first implemented DeepSimulator using 5mer. When we used 5mer, the context-dependent pore model is significantly better than the context-independent pore morel, as illustrated in the paper.
2. Later on, Nanoporetech released the official 6mer table. Compared to 5mer, 6mer is more precise. When compared with the 6mer context-independent pore model, the advantage of the context-dependent pore model is not that significant. It is reasonable since longer the element is, the more information it will contain. Furthermore, the Nanopore sequencing itself does not rely on very long range information.
3. The most important reason is that the context-independent pore model is faster than the context-dependent pore model. It may take the users a very long time to run against the whole genome.

Based on the above consideration, we set the default pore model as the context-independent 6mer pore model. But we preserved the other pore models as the optional models for the users' choice.
